coalib/bearlib/abstractions/Linter.py

```diff
@@ -447,10 +447,17 @@ def run(self, filename, file, **kwargs):
                     FunctionMetadata.filter_parameters(
                         self._get_create_arguments_metadata(), kwargs))
 
-                arguments = (self.get_executable(),) + tuple(
-                    self.create_arguments(
-                        filename, file, config_file,
-                        **create_arguments_kwargs))
+                args = self.create_arguments(filename, file, config_file,
+                                             **create_arguments_kwargs)
+
+                try:
+                    args = tuple(args)
```

Comment:
it's good that it wasn't inlined, if `create_argument` raises a TypeError this would be catched too, but in this case the bear shall crash^^